addons/im_livechat/models/discuss_channel.py

```diff
@@ -168,8 +192,9 @@ def _get_channel_history(self):
         """
         Converting message body back to plaintext for correct data formatting in HTML field.
         """
+        # sudo - res.partner: public user can access internal user messages
```

Comment:
Why is this needed now?

Reply:
We need it so that public user will be able to access internal user's data while creating a ticket.

Reply:
Then it should be fixed before this PR right? I mean its part of a chat bot flow that existed before your PR isn't it?

Reply:
Before this PR there wouldn't be any scenario where it would have needed to access any Internal user's data. So, this bug will be introduced by this PR. So, I think we should keep it in this PR itself.

Maybe in a separate commit?

Reply:
Oh, I get it, before this PR there wouldn't be any operator in the channel so this was not needed. That's fine then, thanks for the explanation :)